src/app/create/codebases/codebases-item-actions/codebases-item-actions.component.scss

```diff
@@ -1,2 +1,9 @@
 @import '../../../../assets/stylesheets/base';
-codebases-item-actions { display: flex; }
+
+codebases-item-actions {
+  display: flex;
```

Comment:
Based on the standard that @mindreeper2420 has started us using this should be on one line if there is only one attribute.